Automattic/wp-calypso
# Stats: Swapping Authors, Click and Referrers components

#### Proposed Changes

* using a feature flag and replacing `Authors`, `Clicks` and `Referrers` StatsModules with the recently created component with horizontal bars
* extending horizontal bar component to render groups of items

#### Testing Instructions



* Navigate to the main Stats page either on a local instance or the test URL from the comment below
* append `?flags=stats/new-stats-module-component` to the URL
* verify that `Authors` component has horizontal bars
* verify that `Clicks` component has horizontal bars
* verify that `Referrers` component has horizontal bars

#### Pre-merge Checklist






Related to #69044
